tasks/winbuildscripts/Update-Winget.ps1

```diff
@@ -1,7 +1,10 @@
 $ErrorActionPreference = 'Stop';
 Set-Location c:\mnt
 
-(New-Object System.Net.WebClient).DownloadFile("https://github.com/microsoft/winget-create/releases/download/v0.4.0.3-preview/wingetcreate.exe", "$(Get-Location)\wingetcreate.exe")
+$WINGET_DOWNLOAD_URL = if ($env:GENERAL_ARTIFACTS_CACHE_BUCKET_URL) {"${env:GENERAL_ARTIFACTS_CACHE_BUCKET_URL}/winget"} else {"https://github.com/microsoft/winget-create/releases/download"}
```

Comment:
This could be moved to the builder image, so we don't need to download it each time.